docker-java/src/test/java/com/github/dockerjava/junit/DockerCmdExecFactoryDelegate.java

```diff
@@ -24,4 +28,15 @@ public void init(DockerClientConfig dockerClientConfig) {
             ((DockerClientConfigAware) delegate).init(dockerClientConfig);
         }
     }
+    @Override
+    public ListPluginsCmd.Exec listPluginsCmdExec() { return delegate.listPluginsCmdExec(); }
```

Comment:
Please fix the code style here and in other places:
```suggestion
    public ListPluginsCmd.Exec listPluginsCmdExec() {
        return delegate.listPluginsCmdExec();
    }
```